tutorials/25_Customizing_Agent.ipynb

```diff
@@ -0,0 +1,4854 @@
+{
```

Comment:
<p>I would have the 'Keep in mind' note at the top and also add: 'here we show an example of how you might create your own resolver for your custom usecase. Also, if they are defineable, maybe list some example use cases that we already have resolvers for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>